Documentation/Signoffs/Extra.md

```diff
@@ -12,44 +12,41 @@ Figure 1: Extra subsystem and the pause switch can be seen in this figure.
 | --- | --- | --- |
 | 1. |  The interceptor shall have a switch that sets the system into a pause state that will keep the interceptor from firing. | Rulebook |
 | 2. | The voltage switched by the pause switch shall be 5V. | System Constraint|
-| 3. | The interceptor must have lights that are bright enough to be seen by the judges approximately six and a half feet away, which is the longest length of the gameboard. These lights must also be powered by a 5V source from the processor block and must emit a light that falls within the visual light spectrum of 380 to 720 nanometers [1]. | Rulebook / System |
-| 4. | The interceptor must make sounds before firing. The sounds will be powered by a voltage of 5V and the sounds will need to fall within the range of frequency humans can hear which is 20 to 20,000Hz [2]. The volume, or loudness, must fall within a range of 0, the threshold, to a maximum of 120dB and must not exceed 85dB for more than eight hours or it may cause hearing damage [2]. The buzzer should be louder than average room noise which is 30-50 dB [2]. | Rulebook / Safety |
+| 3. | The interceptor must have lights that are bright enough to be seen by the judges approximately six and a half feet away, which is the longest length of the gameboard. Therefore the lights must have a candela rating greater than that of a fire alarm which is 15 cd [1]. These lights must also emit a light that falls within the visual light spectrum of 380 to 720 nanometers [2]. | Rulebook / System |
+| 4. | The interceptor must make sounds before firing. The sounds will need to fall within the range of frequency humans can hear which is 20 to 20,000Hz [3]. The volume, or loudness, must fall within a range of 30-50 dB, which is the average noise of a room [3], to a maximum of 120dB and must not exceed 85dB for more than eight hours or it may cause hearing damage [3]. | Rulebook / Safety |
```

Comment:
Why this range of db? This suggests that you wouldn't hear the siren for the room noise. Think about the logic you applied to the lighting and apply similar logic here.